js/src/ui/Balance/balance.js

```diff
@@ -16,31 +16,46 @@
 
 import BigNumber from 'bignumber.js';
 import React, { Component, PropTypes } from 'react';
+import { FormattedMessage } from 'react-intl';
 import { connect } from 'react-redux';
 
-import unknownImage from '../../../assets/images/contracts/unknown-64x64.png';
+import unknownImage from '~/../assets/images/contracts/unknown-64x64.png';
+
 import styles from './balance.css';
 
 class Balance extends Component {
   static contextTypes = {
     api: PropTypes.object
-  }
+  };
```

Comment:
It should complain about missing semi colons. Not sure why it doesn't in those cases. (And since it hasn't been complaining, I've been very lax - not caring too much one way or another)

There may be a class-based rule, will keep it in mind for the next ESLint update.